packages/@coorpacademy-components/src/molecule/bullet-point-menu-button/index.tsx

```diff
@@ -15,7 +15,19 @@ const BulletPointMenuButton = (props: BulletPointMenuButtonProps) => {
     menuButtonClassName,
     isBulkMenu
   } = props;
-  const handleOnClick = useCallback(() => onClick(), [onClick]);
+  const [visible, setVisible] = useState(false);
+
+  const handleOnClick = useCallback(() => {
+    onClick();
+    setVisible(true);
+  }, [onClick]);
+
+  useEffect(() => {
+    document.addEventListener('mousedown', () => setVisible(false));
+    return () => {
+      document.removeEventListener('mousedown', () => setVisible(false));
+    };
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
      const handleMouseDown = () => setVisible(false);

    document.addEventListener('mousedown', () => handleMouseDown);
    return () => {
      document.removeEventListener('mousedown', () => handleMouseDown);
    };
  }, []);
```

to be sure to remove the function